app/controllers/manage_metadata_controller.rb

```diff
@@ -1,20 +1,7 @@
 # :nodoc:
 class ManageMetadataController < PagesController
-  include BulkUpdates
   include ManageMetadataHelper
 
-  layout 'manage_metadata'
+  # layout 'manage_metadata'
```

Comment:
The layout is being used but not within this controller. Two new controllers have been created for collections and variables where it will be used. This controller is just a container for collections and variables now.